tests/unit/styleMixTests.cpp

```diff
@@ -13,6 +13,54 @@
 using namespace Tangram;
 using YAML::Node;
 
+TEST_CASE( "Style Mixing Test: Nested Style Mixin Nodes", "[mixing][core][yaml]") {
+    SceneLoader sceneLoader;
+    Mixes mix;
+    Node mixNode;
+    Node node = YAML::Load(R"END(
```

Comment:
I would add tests where the styles are defined out of dependency order? I know order is not supposed to be guaranteed, but I had a case in JS where mixing only worked in linear order, but that order was happening most of the time.